### DISCUSSION TITLE
"Error" with  docker compose up -d && docker compose logs -f

### DISCUSSION THREAD

**Discussion by seh2000** [alexjustesen/speedtest-tracker#1947] [Q&A] [answered]
Hi,
Not sure why, but today when updating to latest version I used this docker command `docker compose up -d && docker compose logs -f` which resulted in the below "error" I am not sure what this mean and if it is a error at all, because speedtest-tracker runs like before.
Many thanks in advance.
Steen
 

db-1               | Version: '11.6.2-MariaDB-ubu2404'  socket: '/run/mysqld/mysqld.sock'  port: 3306  mariadb.org binary distribution
db-1               | 2024-12-20 18:29:13 3 [Warning] Aborted connection 3 to db: 'unconnected' user: 'unauthenticated' host: '172.19.0.3' (This connection closed normally without authentication)
db-1               | 2024-12-20 18:32:02 7 [Warning] Aborted connection 7 to db: 'speedtest_tracker' user: 'speedtest_tracker' host: '172.19.0.3' (Got an error reading communication packets)
db-1               | 2024-12-20 18:32:08 88 [Warning] Aborted connection 88 to db: 'unconnected' user: 'unauthenticated' host: '172.19.0.3' (This connection closed normally without authentication)
speedtest-tracker  | speedtest-tracker: https://github.com/sponsors/alexjustesen
speedtest-tracker  |
speedtest-tracker  | To support LSIO projects visit:
speedtest-tracker  | https://www.linuxserver.io/donate/
speedtest-tracker  |
speedtest-tracker  | ───────────────────────────────────────
speedtest-tracker  | GID/UID
speedtest-tracker  | ───────────────────────────────────────
speedtest-tracker  |
speedtest-tracker  | User UID:    1000
speedtest-tracker  | User GID:    1000
speedtest-tracker  | ───────────────────────────────────────
speedtest-tracker  | Linuxserver.io version: v1.0.2-ls66
speedtest-tracker  | Build-date: 2024-12-19T02:22:10+00:00
speedtest-tracker  | ───────────────────────────────────────
speedtest-tracker  |
speedtest-tracker  | using keys found in /config/keys
speedtest-tracker  | Waiting for DB to be available
speedtest-tracker  | **** The following active confs have different version dates than the samples that are shipped. ****
speedtest-tracker  | **** This may be due to user customization or an update to the samples. ****
speedtest-tracker  | **** You should compare the following files to the samples in the same folder and update them. ****
speedtest-tracker  | **** Use the link at the top of the file to view the changelog. ****
speedtest-tracker  | ┌────────────┬────────────┬────────────────────────────────────────────────────────────────────────┐
speedtest-tracker  | │  old date  │  new date  │ path                                                                   │
speedtest-tracker  | ├────────────┼────────────┼────────────────────────────────────────────────────────────────────────┤
speedtest-tracker  | │ 2023-04-13 │ 2024-07-16 │ /config/nginx/site-confs/default.conf                                  │
speedtest-tracker  | │ 2023-04-13 │ 2024-05-27 │ /config/nginx/nginx.conf                                               │
speedtest-tracker  | └────────────┴────────────┴────────────────────────────────────────────────────────────────────────┘
speedtest-tracker  | [custom-init] No custom files found, skipping...
speedtest-tracker  | [ls.io-init] done.
speedtest-tracker  | nginx: [warn] the "listen ... http2" directive is deprecated, use the "http2" directive instead in /config/nginx/site-confs/default.conf:7
speedtest-tracker  | nginx: [warn] the "listen ... http2" directive is deprecated, use the "http2" directive instead in /config/nginx/site-confs/default.conf:8

**Comment by alexjustesen** [comment]
Can you also post your env variables, ideally your whole `docker-compose.yml` file.

**Reply by seh2000** [reply]
Hi Alex,
Yes, my bad :)
Cheers Steen
[Speedtest-tracker Config seh 21-12-2024.txt](https://github.com/user-attachments/files/18218868/Speedtest-tracker.Config.seh.21-12-2024.txt)

**Reply by alexjustesen** [reply]
You can remove the following if you're not using a custom ssl cert or passing any specific configs to the container.

```
volumes:
    - /path/to/data:/config
    - /path/to-custom-ssl-keys:/config/keys
```

Also after the initial container creations did you change any usernames or passwords in your environment variables?

**Reply by seh2000** [reply]
Hi Alex,
Removed 
`volumes:`
   `- /path/to/data:/config`
   `- /path/to-custom-ssl-keys:/config/keys`

I still see when I do `docker compose up -d && docker compose logs -f`

db-1               | 2024-12-21 18:06:09 3 [Warning] Aborted connection 3 to db: 'speedtest_tracker' user: 'speedtest_tracker' host: '172.18.0.3' (Got an error reading communication packets)
db-1               | 2024-12-22 11:54:18 918 [Warning] Aborted connection 918 to db: 'speedtest_tracker' user: 'speedtest_tracker' host: '172.18.0.3' (Got an error reading communication packets)
db-1               | 2024-12-22 11:55:04 26681 [Warning] Aborted connection 26681 to db: 'speedtest_tracker' user: 'speedtest_tracker' host: '172.18.0.3' (Got an error reading communication packets)
db-1               | 2024-12-22 12:04:32 26702 [Warning] Aborted connection 26702 to db: 'speedtest_tracker' user: 'speedtest_tracker' host: '172.18.0.3' (Got an error reading communication packets)

But not when I do `docker logs -f speedtest-tracker`

Still all appears to run OK.

<img width="603" alt="image" src="https://github.com/user-attachments/assets/097afbc0-a824-428c-8250-e45c6668134a" />

**Reply by alexjustesen** [reply, marked as answer]
You can probably ignore this as long as everything seems to be working fine.

**Reply by seh2000** [reply]
Hi Alex,
Will do :)
Merry Christmas and a Happy new Year!
